org.spoofax.interpreter.core/src/main/java/org/spoofax/interpreter/library/ssl/SSL_immutable_relation_from_list.java

```diff
@@ -0,0 +1,36 @@
+package org.spoofax.interpreter.library.ssl;
+
+import io.usethesource.capsule.BinaryRelation;
+
+import org.spoofax.interpreter.core.IContext;
+import org.spoofax.interpreter.library.AbstractPrimitive;
+import org.spoofax.interpreter.stratego.Strategy;
+import org.spoofax.interpreter.terms.IStrategoList;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+import org.spoofax.terms.util.TermUtils;
+
+public class SSL_immutable_relation_from_list extends AbstractPrimitive {
+
+    protected SSL_immutable_relation_from_list() {
+        super("SSL_immutable_relation_from_list", 0, 0);
+    }
+
+    @Override
+    public boolean call(IContext env, Strategy[] sargs, IStrategoTerm[] targs) {
+        if(!TermUtils.isList(env.current())) {
+            return false;
```

Comment:
Again a question of failure vs exception. However, in this case it is the input term, and it is more in line with Stratego practice to backtrack on incompatible input. So perhaps here I'd return `false` as well.